plugins/scanners/scanoss/src/main/kotlin/ScanOssResultParser.kt

```diff
@@ -137,16 +151,14 @@ private fun getCopyrightFindings(details: ScanFileDetails): List<CopyrightFindin
  * Get the snippet findings from the given [details]. If a snippet returned by ScanOSS contains several Purls,
  * several snippets are created in ORT each containing a single Purl.
  */
-private fun getSnippets(details: ScanFileDetails): Set<Snippet> {
+private fun getSnippets(details: ScanFileDetails): List<Snippet> {
```

Comment:
Why does this need to return a list now?

Reply:
We changed the return type from Set to List because we need to maintain a consistent ordering of snippets to ensure we always select the first PURL as the primary identifier